docs/extending_filer.rst

```diff
@@ -0,0 +1,342 @@
+.. _extending_filer:
```

Comment:
Can you just add in the additional changes that this PR adds rather than the whole file from filer? This makes updating filer easier in the future.

Reply:
Fixed